src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -1293,9 +1326,36 @@
             return dayNamesHtml;
         },
 
+        calculateWeekNumber_iso8601: function(d) {
+            d = new Date(Date.UTC(d.year, d.month, d.day));
+            d.setUTCDate(d.getUTCDate() + 4 - (d.getUTCDay()||7));
+            return calculateWeekNumber(d);
+        },
+
+        calculateWeekNumber_sundayFirst: function (d) {
+            d = new Date(Date.UTC(d.year, d.month, d.day));
+            d.setUTCDate(d.getUTCDate() + 3 - d.getUTCDay());
+            return calculateWeekNumber(d);
+        },
+
+        calculateWeekNumber_saturdayFirst: function (d) {
+            d = new Date(Date.UTC(d.year, d.month, d.day));
+            d.setUTCDate(d.getUTCDate() - ((d.getUTCDay() + 6) % 7));
+            return calculateWeekNumber(d);
+        },
+
         renderWeek: function (weekDates) {
             var weekHtml = '';
 
+            if(this.options.showWeek) {
+                var date = weekDates[0],
+                    cellClass = date.otherMonth && !this.options.showOtherMonths ? ' ui-datepicker-other-month-hidden' : '';
+
+                weekHtml += '<td class="ui-datepicker-week-col' + cellClass + '">' + 
```

Comment:
please add ui-state-disabled

Reply:
I have implemeted it the same ways a jquery did it. What is your intention adding this CSS class?` When I add `ui-state-disabled`, selecting of the week number is still allowed but it is/may b rendered nearly unreadable (due to some disabled styling like opacity)

Reply:
IMO we should use PrimeNG and PrimeReact as reference.
https://primefaces.org/primeng/showcase/#/calendar
https://primefaces.org/primereact/showcase/#/calendar
Both use `ui-state-disabled`. 

As far as i know PrimeTek uses the (almost) same base SASS for all themes independent of technology. So it would make sense to render the same code like PrimeNG/PrimeReact/PrimeVue.
@mertsincan: Do you want to add something?

Reply:
Hi,

DOM structure can be like; 
```
<td class="ui-datepicker-weeknumber">
     <span class="ui-state-disabled">13</span>
</td>
```

Reply:
OK, I will change the DOM structure as requested.